convert.py

```diff
@@ -0,0 +1,604 @@
+from cdp_backend.pipeline import ingestion_models
```

Comment:
Leave this for now, but ultimately we will want to remove this file entirely before merging.